### DISCUSSION TITLE
is there a public daily.dev API?

### DISCUSSION THREAD

**Discussion by Ezar101** [dailydotdev/daily#249] [General]
Hello, I would like to know if there is a public api for daily.dev that returns json of feeds, posts, cats, ...etc, if so what's the entrypoint?
Thanks

**Comment by idoshamun** [comment]
Hey @Ezar101,

At the moment we don't offer a public API for daily.dev

**Reply by Ezar101** [reply]
thanks @idoshamun  for answer

**Reply by hitesh-55** [reply]
Hey is there any update on this ?

**Comment by nimrodkra** [comment]
We don’t have a public API available for daily.dev. If this changes in the
future, we’ll be sure to share updates with the community.

Out of curiosity, what use case do you have in mind? What do you want to
build with it?

**Reply by Ezar101** [reply]
Wow, this post is so old.
Hello @nimrodkra 
Actually, at that time, there wasn't a mobile app dedicated to Daily.dev yet, and I wanted to develop one, but today it doesn't matter.
thanks

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
